client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
```

Comment:
remember to change the cities we wanted based on our conversation during the standup